aeproject-cli/aeproject-init/artifacts/exampleTests.js

```diff
@@ -0,0 +1,77 @@
+/*
+ * ISC License (ISC)
+ * Copyright (c) 2018 aeternity developers
+ *
+ *  Permission to use, copy, modify, and/or distribute this software for any
+ *  purpose with or without fee is hereby granted, provided that the above
+ *  copyright notice and this permission notice appear in all copies.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH
+ *  REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY
+ *  AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT,
+ *  INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM
+ *  LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR
+ *  OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
+ *  PERFORMANCE OF THIS SOFTWARE.
+ */
+const fs = require('fs');
+const chai = require('chai');
+var chai_as_promised = require("chai-as-promised");
+chai.use(chai_as_promised);
+const assert = chai.assert;
+const expect = chai.expect;
+
+const { Universal, MemoryAccount, Node } = require('@aeternity/aepp-sdk');
+
+const EXAMPLE_CONTRACT = fs.readFileSync('./contracts/ExampleContract.aes', 'utf8');
+const NETWORKS = require('../config/network.json');
```

Comment:
why move that into every project instead of aeproject dependency?

Reply:
mhh, as devs install aeproject globally to use the CLI I wanted to avoid having to add aeproject in each project, too. this way it can also be modified easily. not sure what's better here

Reply:
as discussed I will leave that as is right now. certain files and functions will be copied into every new project for the moment. I also added contract-utils that can determine the `filesystem` object which needs to be provided to the compiler for custom includes. the provided example now also contains a custom include.